spec/views/idv/hybrid_handoff/show.html.erb_spec.rb

```diff
@@ -12,57 +12,24 @@
     }
   end
 
-  context 'with show phone question' do
-    before do
-      @phone_question_ab_test_bucket = :show_phone_question
-    end
-
-    it 'has a form for starting mobile doc auth with an aria label tag' do
-      expect(rendered).to have_selector(
-        :xpath,
-        "//form[@aria-label=\"#{t('forms.buttons.send_link')}\"]",
-      )
-    end
-
-    it 'has a form for starting desktop doc auth with an aria label tag' do
-      expect(rendered).to have_selector(
-        :xpath,
-        "//form[@aria-label=\"#{t('forms.buttons.upload_photos')}\"]",
-      )
-    end
-
-    it 'displays the expected headings from the "b" case' do
-      expect(rendered).to have_selector('h1', text: t('doc_auth.headings.upload_from_phone'))
-      expect(rendered).to have_selector('h2', text: t('doc_auth.headings.switch_to_phone'))
-      expect(rendered).not_to have_selector('h1', text: t('doc_auth.headings.hybrid_handoff'))
-      expect(rendered).not_to have_selector('h2', text: t('doc_auth.headings.upload_from_phone'))
-    end
+  it 'has a form for starting mobile doc auth with an aria label tag' do
+    expect(rendered).to have_selector(
+      :xpath,
+      "//form[@aria-label=\"#{t('forms.buttons.send_link')}\"]",
+    )
   end
 
-  context 'without show phone question' do
-    before do
-      @phone_question_ab_test_bucket = :bypass_phone_question
-    end
-
-    it 'has a form for starting mobile doc auth with an aria label tag' do
-      expect(rendered).to have_selector(
-        :xpath,
-        "//form[@aria-label=\"#{t('forms.buttons.send_link')}\"]",
-      )
-    end
-
-    it 'has a form for starting desktop doc auth with an aria label tag' do
-      expect(rendered).to have_selector(
-        :xpath,
-        "//form[@aria-label=\"#{t('forms.buttons.upload_photos')}\"]",
-      )
-    end
+  it 'has a form for starting desktop doc auth with an aria label tag' do
+    expect(rendered).to have_selector(
+      :xpath,
+      "//form[@aria-label=\"#{t('forms.buttons.upload_photos')}\"]",
+    )
+  end
 
-    it 'displays the expected headings from the "a" case' do
-      expect(rendered).to have_selector('h1', text: t('doc_auth.headings.hybrid_handoff'))
-      expect(rendered).to have_selector('h2', text: t('doc_auth.headings.upload_from_phone'))
-      expect(rendered).not_to have_selector('h1', text: t('doc_auth.headings.upload_from_phone'))
-      expect(rendered).not_to have_selector('h2', text: t('doc_auth.headings.switch_to_phone'))
-    end
+  it 'displays the expected headings from the "a" case' do
+    expect(rendered).to have_selector('h1', text: t('doc_auth.headings.hybrid_handoff'))
+    expect(rendered).to have_selector('h2', text: t('doc_auth.headings.upload_from_phone'))
+    expect(rendered).not_to have_selector('h1', text: t('doc_auth.headings.upload_from_phone'))
+    expect(rendered).not_to have_selector('h2', text: t('doc_auth.headings.switch_to_phone'))
   end
```

Comment:
Do we still need this spec? It looks like it's related to the a/b test. It might need an edited title.